airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py

```diff
@@ -247,6 +273,18 @@ class AdGroupAds(IncrementalGoogleAdsStream):
     primary_key = ["ad_group_ad.ad.id", "segments.date"]
 
 
+class AdGroupAdLabels(GoogleAdsStream):
+    """
+    Ad Group Ad Labels stream: https://developers.google.com/google-ads/api/fields/v8/ad_group_ad_label
+    """
+
+    def __init__(self, api):
```

Comment:
fwiw in python you don't need a constructor for a subclass if the constructor exactly matches that of the superclass. You can just get rid of all these constructors and it would work the same